src/__tests__/Search.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import React from 'react';
+import store from '../Redux/configureStore';
+import Search from '../Components/Search';
+
+describe('Search component', () => {
+  test('Renders Search component', () => {
+    const tree = render(
+      <React.StrictMode>
+        <Provider store={store}>
+          <Router>
+            <Search />
+          </Router>
+        </Provider>
+      </React.StrictMode>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] good job testing your components 👏 but as part of the requirements kindly add some unit & integration tests for some pure functions in your app 🙏

Reply:
thank you